packages/dd-trace/src/appsec/iast/taint-tracking/operations.js

```diff
@@ -96,12 +111,16 @@ function getRanges (iastContext, string) {
   return result
 }
 
-function enableTaintOperations () {
-  global._ddiast = TaintTracking
+function enableTaintOperations (telemetryVerbosity) {
+  if (isInfoAllowed(telemetryVerbosity)) {
+    onRemoveTransaction = onRemoveTransactionInformationTelemetry
+  }
+
+  global._ddiast = getTaintTrackingImpl(telemetryVerbosity)
 }
 
 function disableTaintOperations () {
-  global._ddiast = TaintTrackingDummy
+  global._ddiast = getTaintTrackingDummy()
```

Comment:
`getTaintTrackingNoop()` instead of `getTaintTrackingDummy()`? It is more used, for example in Proxy and NoopProxy.